packages/devextreme/js/__internal/ui/date_range_box/m_date_range_box.ts

```diff
@@ -149,7 +149,7 @@ class DateRangeBox extends Editor<Properties> {
       value: [null, null],
       valueChangeEvent: 'change',
       _internalValidationErrors: [],
-      _currentSelection: 'startDate',
```

Comment:
why if we don't publish it?

Reply:
> we alredy strarted to name private options without _ prefix as well
Didn't know about this 🤔
Maybe you're about private class props not options? It sounds more reasonable because modificators can be used. But for options I would prefer to differ private and public ones, don't u?